process/sync/baseSync.go

```diff
@@ -733,14 +733,13 @@ func (boot *baseBootstrap) rollBack(revertUsingForkNonce bool) error {
 
 		err = boot.historyRepo.RevertBlock(currHeader, currBody)
 		if err != nil {
-			log.Debug("boot.historyRepo.RevertBlock",
-				"error", err.Error(),
-			)
-
 			return err
 		}
 
-		boot.outportHandler.RevertIndexedBlock(currHeader, currBody)
+		err = boot.outportHandler.RevertIndexedBlock(currHeader, currBody)
+		if err != nil {
+			return err
```

Comment:
I think that this return and also the one from line 736 should be avoided. Maybe only log.Debug with error, as the rollback has been already done in L713, and after this line the code should not return err, just for the consistency.